README.md

```diff
@@ -58,6 +59,25 @@ epsagon.Label('items_in_cart', items_in_cart)
 
 Valid types are `string`, `bool`, `int` and `float`.
 
+
+### Custom Errors
+
+You can set a trace as an error (although handled correctly) to get an alert or just follow it on the dashboard.
```

Comment:
I'd maybe flip the order of the two, just because we want to encourage users to use typed errors